compiler/crates/graphql-syntax/src/node/type_system.rs

```diff
@@ -515,7 +515,7 @@ impl fmt::Display for DirectiveLocation {
 pub struct InputValueDefinition {
     pub name: Identifier,
     pub type_: TypeAnnotation,
-    pub default_value: Option<ConstantValue>,
+    pub default_value: Option<DefaultValue>,
```

Comment:
Is there more context on why it's useful to have a DefaultValue here?

If we want to do this, I think we should move where `DefaultValue` is defined, perhaps into `compiler/crates/graphql-syntax/src/node/constant_value.rs`? Right now it's defined with the executable definition AST constructs, so I don't think it makes sense to share it between both types of documents without moving it.

Reply:
The `ConstantValue` doesn't have span information, so we can't determine whether we are on the default value of a `InputValueDefinition` in the path resolution. The `DefaultValue` has a span definition and made sense to me here, as it's the same concept as a default value for a `VariableDefinition`.

Reply:
Cool. What do you think about moving the definition into constant_value.rs?